irods/test/helpers.py

```diff
@@ -151,6 +151,18 @@ def recast(k):
 # Create a connection for test, based on ~/.irods environment by default.
 
 def make_session(test_server_version = True, **kwargs):
+    '''
+    Connect to an active iRODS server as determined by any client environment
+    file present at a standard location, and by any keyword arguments given.
+
+    Arguments:
+
+    test_server_version (True or False).  Raises iRODS_Server_Too_Recent if
+    the server connected to is more recent than the current python iRODS
+    client's advertised level of compatibility.
+
+    **kwargs: keyword arguments.  Fed directly to the iRODSSession constructor.
```

Comment:
The `test_server_version` parameter name and description isn't separated by a colon like `**kwargs`. Is that intentional?

Reply:
Not intentional, correcting....

Reply:
Done.